Resources/views/CRUD/base_acl_macro.html.twig

```diff
@@ -17,9 +17,12 @@ file that was distributed with this source code.
             {% if not admin_pool.getOption('html5_validate') %}novalidate="novalidate"{% endif %}
             >
         {% if form.vars.errors|length > 0 %}
-            <div class="sonata-ba-form-error">
-                {{ form_errors(form) }}
-            </div>
+            {% for error in form.vars.errors %}
```

Comment:
I don't know why we have an if here… could `errors` be null ? If yes, then `{% for error in form.vars.errors|default([]) %}` could spare us the `length` check